storage/service_account/hmac/hmac_test.go

```diff
@@ -162,13 +162,25 @@ func TestDeleteKey(t *testing.T) {
 }
 
 // Create a key for testing purposes.
-func createTestKey(projectID string) (*storage.HMACKey, error) {
+func createTestKey(projectID string, t *testing.T) (*storage.HMACKey, error) {
 	ctx := context.Background()
-	key, err := storageClient.CreateHMACKey(ctx, projectID, serviceAccountEmail)
-	if err != nil {
-		return nil, fmt.Errorf("storage.NewClient: %v", err)
-	}
-	return key, nil
+	var key *storage.HMACKey
+	var err error
+
+	testutil.Retry(t, 10, 10*time.Second, func(r *testutil.R) {
+		key, err = storageClient.CreateHMACKey(ctx, projectID, serviceAccountEmail)
+		if err != nil {
+			t.Fatalf("Error in CreateHMACKey: %v", err)
```

Comment:
As @tbpg said, this and L178 should be `r.Errorf`. Otherwise the retry will not work.

Reply:
yep, just fixed this in latest commit